OrangeDesignSystemDemo/OrangeDesignSystemDemo/Screens/Modules/Lists/ListModuleDataModel.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Software Name: Orange Design System (iOS)
+// SPDX-FileCopyrightText: Copyright (c) 2021 - 2023 Orange SA
+// SPDX-License-Identifier: MIT
+//
+// This software is distributed under the MIT license.
+//
+
+import SwiftUI
+
+class ListModuleDataModel: ObservableObject {
+
+    // =======================
+    // MARK: Stored properties
+    // =======================
+
+    @Published var selectedFoods: [Food] {
+        didSet { updateRecipes() }
```

Comment:
Are you sure about the performances? Assigning **selectedFoods** will redraw the view, then update **filetedReceipes** which will redraw the view again.